src/plugins/dashboard/public/plugin.tsx

```diff
@@ -342,6 +359,32 @@ export class DashboardPlugin
     }
   }
 
+  private navigateToDashboard(core: CoreStart, dashInput: EmbeddableInput) {
+    if (!this.getActiveUrl) {
```

Comment:
should we navigate to empty dashboard if one was not loaded ? should we allow for passing dashboardId to navigateToDashboard function ?